cli/ps.go

```diff
@@ -88,24 +94,26 @@ func (p *PsCommand) runPs(args []string) error {
 		}
 		return nil
 	}
-
-	display := p.cli.NewTableDisplay()
-	display.AddRow([]string{"Name", "ID", "Status", "Created", "Image", "Runtime"})
-
+	// add to format the output with go template
+	format := p.flagFormat
```

Comment:
I think we can refactor code to include the quiet mode. I am fine to defer to discuss with this.

Reply:
if it is quiet mode, we have no need to care about the format option.